test/Test/Database/LSMTree/Internal/Snapshot/Codec.hs

```diff
@@ -47,6 +49,7 @@ tests = testGroup "Test.Database.LSMTree.Internal.Snapshot.Codec" [
         testAll $ \(p :: Proxy a) ->
           testGroup (show $ typeRep p) $
             prop_arbitraryAndShrinkPreserveInvariant @a deepseqInvariant
+    , goldenFileTests
```

Comment:
The typical approach we use in this test suite is to include the tests at the top level, so in the `Main` module